### ISSUE TITLE
Add MobileNetV2 to torchvision

### ISSUE THREAD

**Issue by tonylins** [pytorch/vision#625]
Hi,

I have implemented MobileNetV2 with PyTorch and successfully reproduced the results here: https://github.com/tonylins/pytorch-mobilenet-v2

The repo has received more than 300 stars right now. I'm wondering if it is possible to merge the model into torchvision so that more people can benefit from it.

Thanks so much!

**Comment by fmassa** [comment]
Yes, it will be a great contribution to torchvision.

Did you train your model using the standard `pytorch/examples/imagenet` codebase? What were the modifications that you did?

**Comment by tonylins** [comment]
Thanks! Yes, I used the standard pytorch ImageNet example codebase.

**Comment by fmassa** [comment]
What about the learning rate schedules / number of epochs, was it the same?

**Comment by tonylins** [comment]
No, I followed the training protocol in the paper. The initial learning rate was 0.045, and decayed by 0.94 every 2 epochs.

**Comment by fmassa** [comment]
Sounds good.

Would you be willing to send a PR with the model? And also with a link to the pre-trained weights and the script that you used to train it (in a gist for example) would be awesome!

**Comment by fmassa** [comment]
Fixed via #818
